CodeEditModules/Modules/Git/src/Client/Models/ChangedFiles.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ChangedFiles.swift
+//  
+//
+//  Created by Nanashi Li on 2022/05/20.
+//
+
+import Foundation
+import SwiftUI
+import WorkspaceClient
+
+public struct ChangedFiles: Codable, Hashable, Identifiable {
```

Comment:
I believe this struct is only taking one file per object. It should be clear if we change it to the singular, `ChangedFile`.

There are a lot of places using this struct, which may cause the confusion.
You may use refactor -> rename this struct so you can easily change all usages of this struct.